packages/core/middlewares/errors.js

```diff
@@ -1,12 +1,19 @@
+// @ts-check
+
+/** @type {import('../types/index.d.ts').TrifidMiddleware} */
 const factory = (trifid) => {
-  const { logger } = trifid;
+  const { logger } = trifid
 
   return (err, _req, res, _next) => {
-    logger.error(err.stack);
+    logger.error(err.stack)
+
+    res.statusCode = res.statusCode || 500
+    if (res.statusCode < 400) {
+      res.statusCode = 500
+    }
```

Comment:
Would this code be cleaner with `res.sendStatus(number)` instead of assigning `statusCode`?

Reply:
Also, I would suggest adding `Problem+JSON`, for example using [express-rdf-problem-details](https://www.npmjs.com/package/express-rdf-problem-details) which responds with errors as JSON-LD documents

Reply:
> Would this code be cleaner with `res.sendStatus(number)` instead of assigning `statusCode`?

Done. But it has the side effect that the body will be populated with a description of the error message. I needed to adjust a test, and I added an entry to the changelogs to make it explicit.


> Also, I would suggest adding `Problem+JSON`, for example using [express-rdf-problem-details](https://www.npmjs.com/package/express-rdf-problem-details) which responds with errors as JSON-LD documents

Thank you for the suggestion. This can be an idea for the future, but it's not something I want to introduce for now.